From: =?utf-8?q?Fabian_Gr=C3=BCnbichler?= <git@fabian.gruenbichler.email>
Date: Sat, 20 Sep 2025 15:56:29 +0200
Subject: Revert "Enable `f16` for MIPS"

This reverts commit cf1242c461e035fd2cfb99f620e810015744cac4.

Description: requires LLVM 20
---
 library/std/build.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/library/std/build.rs b/library/std/build.rs
index fe47088..c467980 100644
--- a/library/std/build.rs
+++ b/library/std/build.rs
@@ -116,6 +116,7 @@ fn main() {
         ("csky", _) => false,
         ("hexagon", _) => false,
         ("loongarch64", _) => false,
+        ("mips" | "mips64" | "mips32r6" | "mips64r6", _) => false,
         ("powerpc" | "powerpc64", _) => false,
         ("sparc" | "sparc64", _) => false,
         ("wasm32" | "wasm64", _) => false,
